package.json

```diff
@@ -23,6 +23,14 @@
     "url": "https://github.com/City-of-Helsinki/kerrokantasi-ui/issues"
   },
   "dependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/register": "^7.0.0",
```

Comment:
I'm having this error when trying to run build: `Error: Cannot find module 'babel-register'`, not sure why that is 🤔

Reply:
now I got same problem. I was pretty sure I fixed this already. Now I can't find a sollution.

Reply:
I think I found it in `compile/index.js`. Need to replace `require('babel-register');` -> `require('@babel/register');`

Reply:
yes, that was it